src/utils/common.js

```diff
@@ -1,3 +1,12 @@
 export function isInBrowser() {
   return typeof window !== 'undefined';
 }
+
+export function replaceSpaces(str) {
+  return str.replace(/ /, '-');
+}
+
+export function getAnchorName(arr) {
+  // ['anna luise', 'jones'] => `anna-luise-jones`
+  return arr.map(el => replaceSpaces(el)).join('-');
+}
```

Comment:
`replaceSpaces` isn't used anywhere except one place, right? first no need to export and second no need for that function.

and no need in `getAnchorName`